### PR TITLE
Add _afterPaintQueued and _fallback to mangle.json

### PR DESCRIPTION
**Pull request by 38elements**

### CODE REVIEW

File: hooks/mangle.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "props": {
     "cname": 6,
     "props": {
+      "$_afterPaintQueued": "__a",
```
Comment (38elements) [inline hooks/mangle.json]: https://github.com/preactjs/preact/blob/0aeca9ac0bc089d5df4549b1ef54a15de9f135a8/hooks/src/index.js#L220

File: compat/mangle.json [hunk 2]
```diff
@@ -20,6 +20,7 @@
       "$_root": "__p",
       "$_commit": "__c",
       "$_suspensions": "__u",
+      "$_fallback": "__",
```
Comment (38elements) [inline compat/mangle.json]: https://github.com/preactjs/preact/blob/b051204ca2dbcd409288a6a74d1ec8fbeae94fd9/compat/src/suspense.js#L46

Comment (38elements) [reply compat/mangle.json]: @JoviDeCroock 
Thank you for review.
I am sorry.
I changed this line for #2044.

File: compat/mangle.json [hunk 3]
```diff
@@ -20,11 +20,12 @@
...
       "$_diff": "__b",
       "$_render": "__r",
       "$_hook": "__h",
       "$_catchError": "__e",
       "$_unmount": "_e"
     }
   }
-}
\ No newline at end of file
+}
```
Comment (andrewiggins) [inline compat/mangle.json]: Nit: This file shouldn't end in a new line. One of the build scripts writes this file after every build and writes it without a final new line. Build all again and you should see a change that removes this final new line (at least it does on my machine 🤷‍♂ ).

Comment (38elements) [reply compat/mangle.json]: @andrewiggins
Thank you for review. 
I removed the last empty line.

**Comment by 38elements** [comment]
@JoviDeCroock @andrewiggins 
I am sorry.
I removed the end line by the editor of Github.
But, the end line was not removed. 
I removed it at #2053.

**Comment by andrewiggins** [comment]
No worries! Thanks for the PRs!

**Pull request merged**